components/global/core/io.cellery.observability.api/src/main/java/io/cellery/observability/api/auth/OIDCOauthManager.java

```diff
@@ -112,24 +106,17 @@ private JSONObject getClientCredentials() throws OIDCProviderException {
             return jsonObject;
         } catch (NoSuchAlgorithmException | KeyManagementException | ConfigurationException | IOException e) {
             throw new OIDCProviderException("Error occured while registering client", e);
-        } finally {
-            if (bufReader != null) {
-                try {
-                    bufReader.close();
-                } catch (IOException e) {
-                    log.debug("Error occured while closing buffered reader ", e);
-                }
-            }
-            if (inputStreamReader != null) {
-                try {
-                    inputStreamReader.close();
-                } catch (IOException e) {
-                    log.debug("Error occured while closing input stream reader ", e);
-                }
-            }
         }
     }
 
+    /**
+     * This method will be used to construct request body for DCR registeration
+     *
+     * @param dcrEp        This will be the DCR URL.
```

Comment:
Don't add "." at the end of the param and return description. Also add a "." at the end of the method description.